src/devices/Card/Ultralight/MirrorConfiguration.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Card.Ultralight
+{
+    /// <summary>
+    /// The Mirror Configuration
+    /// </summary>
+    public class MirrorConfiguration
+    {
+        /// <summary>
+        /// The size in bytes of the UID Mirror
+        /// </summary>
+        public const byte UidMirrorSize = 14;
```

Comment:
should these be internal?

Reply:
No as you need to know what's the size. You can create an NDEF message like:
`My card number is 00000000000000 and it's cool` (I'm using 0 but it can be any character, not even the same one)
It will ben be transformed automatically into:
`My card number is 04ABCDEF123456 and it's cool`
So it's about exposing the size of the Mirror UID, same for the other one